native/packages/react-native-bpk-component-picker/src/BpkPickerMenu.android.js

```diff
@@ -27,64 +33,109 @@ import {
   lineHeightBase,
   spacingBase,
 } from 'bpk-tokens/tokens/base.react.native';
+import BpkPickerItem from './BpkPickerItem';
 
-const rowsToDisplay = 6;
-
-const maxListHeight =
-  // eslint-disable-next-line no-mixed-operators
-  (2 * spacingBase + lineHeightBase) * rowsToDisplay;
+const MAX_ROWS_TO_DISPLAY = 6;
 
 const styles = StyleSheet.create({
   overlay: {
-    flex: 1,
+    left: 0,
+    height: '100%',
+    position: 'absolute',
+    top: 0,
+    width: '100%',
     backgroundColor: setOpacity(colorGray900, 0.8),
-    justifyContent: 'center',
   },
   list: {
     backgroundColor: colorWhite,
     borderRadius: borderRadiusSm,
-    maxHeight: maxListHeight,
-    margin: spacingBase,
-    flex: 0,
+    alignSelf: 'center',
+    top: '50%',
```

Comment:
Isn't flexbox much better suited to vertically centering things?

Reply:
I struggled to do it in the Y-direction - wanna pair on trying to get it working? 🙏 📿